public-ui/src/app/data-browser/charts/chart-survey-answers/chart-survey-answers.component.css

```diff
@@ -0,0 +1,26 @@
+
+.answer-chart-layout {
+    display:flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+.legend {
+    padding:1rem;
+    padding-bottom: 2rem;
+}
+
+.legend-inner {
+    overflow-y: scroll;
```

Comment:
![image](https://user-images.githubusercontent.com/21062724/99565211-ffe29180-2990-11eb-9638-95320b0b819a.png)
Can this be set to auto to not have the scroll bar when not needed ?

Reply:
thanks. I will fix that in the tool tip PR